CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.41.12] - 2023-03-27
+Encode PrimitiveSchema's properties in PdlEncoder
```

Comment:
this is a gap. We previously  don't encode a property along with primitive data schema.  Otherwise how are you going to encode a property to the primitive schema